### PR TITLE
Post update delete and log out

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check 
`src/Redux/Reducers/authSlice.js`
`src/Redux/Reducers/postsSlice.js`
`src/components/modal/Modal.jsx`
`src/pages/feeds/Feeds.jsx`

## [Live](https://share-arts-git-post-update-delete-developers-codz.vercel.app/)

Thank you for reviewing my PR :)

### CODE REVIEW

File: src/Redux/Reducers/postsSlice.js [hunk 1]
```diff
@@ -1,137 +1,237 @@
...
+        state.posts = action.payload.posts;
+      })
+      .addCase(deletePost.rejected, (state, action) => {})
+
+      .addCase(editPost.fulfilled, (state, action) => {
+        console.log("jii");
+        state.posts = action.payload.posts;
+      })
+      .addCase(editPost.rejected, (state, action) => {
+        console.log("jii");
```
Comment (hsnice16) [inline src/Redux/Reducers/postsSlice.js]: Are these console statements necessary? If not, then you can remove.

**Review by hsnice16 — APPROVED** [review]
Great work, @Developers-codz 
Have added a suggestion and would also suggest checking for the console statement in the whole PR once. 
Good Job.

**Review by Maianki — APPROVED** [review]
Everything looks good functionality is working fine. Jsut remove logs from code.

**Comment by Developers-codz** [comment]
> Great work, @Developers-codz Have added a suggestion and would also suggest checking for the console statement in the whole PR once. Good Job.

okay

**Pull request merged**